apps/rush-lib/src/logic/buildCache/CacheEntryId.ts

```diff
@@ -84,6 +86,31 @@ export class CacheEntryId {
             }
           }
 
+          case PHASE_NAME_TOKEN_NAME: {
```

Comment:
Is the cache storage format changing in a way that could be incompatible with cache files created by a previous release of Rush?  If so, should we serialize some kind of version so that we can detect this situation and flush the cache?

Reply:
No. Someone will need to explicitly update the `cacheEntryNamePattern` field of their `build-cache.json` file.